plugin/src/main/java/com/denizenscript/denizen/objects/CuboidTag.java

```diff
@@ -1257,6 +1341,14 @@ public static void register() {
         // Supplying a negative input will therefore contract the cuboid.
         // Note that you can also specify a single number to expand all coordinates by the same amount (equivalent to specifying a location that is that value on X, Y, and Z).
         // Not valid for multi-member CuboidTags.
+        // @example
+        // # If "my_cuboid" spans from 10,10,10 to 5,5,5 and gets expanded by 15 (15,15,15),
+        // # "my_expanded_cuboid" will span -10,-10,-10 to 25,25,25
+        // - note <cuboid[my_cuboid].expand[15]> as:my_expanded_cuboid
+        // @example
+        // # If "my_cuboid" spans from 10,10,10 to 5,5,5 and gets expanded by 15,15,15,
+        // # "my_expanded_cuboid" will span -10,-10,-10 to 25,25,25
+        // - note <cuboid[my_cuboid].expand[15,15,15]> as:my_expanded_cuboid
```

Comment:
Should have different numbers in the 3 coordinates for this example, to emphasize what makes it different from the other.